cpp/examples/ide_secir.cpp

```diff
@@ -72,11 +72,13 @@ int main()
     mio::SmootherCosine smoothcos(2.0);
     mio::StateAgeFunctionWrapper delaydistribution(smoothcos);
     std::vector<mio::StateAgeFunctionWrapper> vec_delaydistrib(num_transitions, delaydistribution);
-    vec_delaydistrib[(int)mio::isecir::InfectionTransition::SusceptibleToExposed].set_parameter(3.0);
+    // TransitionDistribution for SusceptibleToExposed is not used.
+    vec_delaydistrib[(int)mio::isecir::InfectionTransition::SusceptibleToExposed].set_parameter(-1.);
```

Comment:
I would either not set this parameter explicitly or comment that this can be set arbitrarily.